app/assets/stylesheets/datatables.css

```diff
@@ -1,3 +1,19 @@
+/* 
```

Comment:
I'm guessing this and the datatables.js come from `jquery-datatables` so we should remove the headers on them.

Reply:
:+1: but it looks like datatables.js is still needing its header removed.